src/main/docker/Dockerfile.legacy-jar

```diff
@@ -21,33 +21,34 @@
 # docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/release-manager-jvm
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 ARG JAVA_PACKAGE=java-11-openjdk-headless
 ARG RUN_JAVA_VERSION=1.3.8
 
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
 
 # Install java and the run-java script
-# Also set up permissions for user `1001`
-RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
+# Also set up permissions for user `1000`
+RUN microdnf install curl wget ca-certificates ${JAVA_PACKAGE} \
     && microdnf update \
     && microdnf clean all \
     && mkdir /deployments \
-    && chown 1001 /deployments \
+    && chown 1000 /deployments \
     && chmod "g+rwX" /deployments \
-    && chown 1001:root /deployments \
+    && chown 1000:root /deployments \
     && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1001 /deployments/run-java.sh \
-    && chmod 540 /deployments/run-java.sh \
+    && chown 1000 /deployments/run-java.sh \
+    && chmod 755 /deployments/run-java.sh \
     && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
 
 # Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 
-COPY --chown=1001 target/quarkus-app /deployments/
+COPY target/lib/* /deployments/lib/
+COPY target/*-runner.jar /deployments/app.jar
 
 EXPOSE 8080
-USER 1001
+USER 1000
 
-ENTRYPOINT [ "/deployments/run-java.sh" ]
\ No newline at end of file
+#ENTRYPOINT [ "/deployments/run-java.sh" ]
```

Comment:
There's no entry point anymore?

Reply:
I will create a separate Dockerfile to be used to build the image when used with jenkins as some changes like this or UID = 1000 are needed

Reply:
We can use the `Dockerfile.jvm` for using with Jenkins, I think we are not using it for other purpose. There is also the `legacy-jar` Dockerfile (old Dockerfile.jvm prior to 1.12 version of Quarkus)

Reply:
Ok, so, we don't need the `wget` for the certificates in these files, only in the Jenkins specific one, right? @cmoulliard

Reply:
Yes. wget is not needed in this case. This is only needed for jenkins jnlp @aureamunoz